src/app/components/publiclists/teamlist/teamlist.component.ts

```diff
@@ -227,10 +237,12 @@ export class TeamlistComponent implements OnInit, OnDestroy {
       this.fetchTeamsPath = 'hosts/challenge_host_team';
       this.createTeamsPath = 'hosts/create_challenge_host_team';
       this.deleteTeamsPath = 'hosts/remove_self_from_challenge_host';
+      this.deleteMembersPath = 'hosts/challenge_host_team/';
       this.fetchMyTeams(this.fetchTeamsPath);
       this.teamCreateTitle = 'Create a New Team';
       this.teamSelectTitle = 'Select a Challenge Host Team';
       this.teamCreateButton = 'Create Host Team';
+      this.MemberDeleteTitle = '/challenge_host/';
```

Comment:
I think this is not an title so how about if we declare only one variable for path to something like this 'hosts/challenge_host_team/<team_id>/challenge_host/' and then [here](https://github.com/Cloud-CV/EvalAI-ngx/pull/288/files?file-filters%5B%5D=.html&file-filters%5B%5D=.ts&file-filters%5B%5D=.yml#diff-cb564c2cd5f0dfd1606b8de1c85f077aR590) we can just replace this `<team_id>` with the team ID. @Kajol-Kumari

Reply:
@Sanji515  as [here](filters%5B%5D=.yml#diff-cb564c2cd5f0dfd1606b8de1c85f077aR259) you can see i am resetting the value of  `MemberDeleteTitle` as this url is being used for both host and participant member deletion so we can't fix the path like this(^).

Reply:
Yes but still I think we can do this way, that single variable will be updated according to the conditions which you have pointed.